ui/static/ui/js/utils.jsx

```diff
@@ -23,6 +23,34 @@ define("utils", ["jquery", "lodash"], function ($, _) {
     });
   };
 
+  /**
+   * Component for message divs. This takes one property. Use like:
+   *   <StatusBox message={{error: "Error message"}} />
+   * or for a regular message:
+   *   <StatusBox message="Hello, world!" />
+   */
+  var StatusBox = React.createClass({
+    render: function() {
+      if (!this.props || !this.props.message) {
+        return null;
+      }
+
+      if (this.props.message.error !== undefined) {
+        return <div className="alert alert-danger alert-dismissible">
+          {this.props.message.error}
+        </div>;
+      }
+
+      if (this.props.message !== undefined) {
+        return <div className="alert alert-success alert-dismissible">
+          {this.props.message}
+          </div>;
+      }
```

Comment:
I see. If you open a learning resource and save the form it should give you the success message there